website_cookiebot/templates/website.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<data>
+    <template id="layout_footer_copyright" inherit_id="web.frontend_layout">
+        <xpath
+            expr="//body"
+            position="inside"
+        >
+            <div>
+                <span t-if="website.cookiebot_dgid">- <a
+                    href="/legal/cookies-policy"
+                >Cookies Policy</a></span>
+            </div>
+
+        </xpath>
+    </template>
```

Comment:
Check this other module implementation that will give you a clue on how to make it better (inherit from that other template): https://github.com/OCA/website/blob/14.0/website_legal_page/views/website_legal_main_page.xml#L4-L18